src/app/AppContainer.js

```diff
@@ -92,7 +91,7 @@ class AppContainer extends React.Component<PropsType, StateType> {
 				toggleHtmlModeAction={ this.toggleHtmlModeAction }
 				setTitleAction={ this.setTitleAction }
```

Comment:
Hey @diegoreymendez 👋 it looks to me that we can also get rid of passing this.setTitleAction as a prop from component to component. So other components can also use editPost( { title: title } ) to via their own props with the help of withDispatch. what do you think?

Reply:
That's a good observation.  Thanks @pinarol!

Considering this PR is working as expected right now though, I've created [an issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/512) that I will be tackling right after this is complete.

As usual my goal is to split the work into atomic steps where necessary.